documentation/tut/toy-example-process.rst

```diff
@@ -0,0 +1,110 @@
+.. _tut_toy_schedule_process:
+
+
+
+Toy example III: Computing schedules for processes
+====================================================================
+
+Until this point we've been using a static battery, one of the most flexible energy assets, to reduce electricity bills. 
+
+However, in some settings, we can reduce electricity bills by *just* consuming energy smartly. In other words, if the process can be displaced, by breaking it into smaller consumption periods or shifting its start time, the process run can match the lower price hours better.
```

Comment:
"by *just* consuming energy smartly." -> 'by *just* smartly timing the necessary work that we know we have to do. We call this work a "process".'

Reply:
Has my suggestion been rejected?

If so, please say so. Better yet of course, give a reason :)

Reply:
Actually added this haha but It got stuck in a stash... thanks for noticing!